src/DataSource.ts

```diff
@@ -35,24 +35,12 @@ export class DataSource extends DataSourceApi<EditorQuery, MyDataSourceOptions>
     /* It runs on 'Test Datasource', and gets the Auth token from Anodot side */
     const defaultErrorMessage = 'Cannot connect to API';
     try {
-      const response = await fetch(this.urlApi + '/access-token', {
-        method: 'POST', // *GET, POST, PUT, DELETE, etc.
-        mode: 'cors', // no-cors, *cors, same-origin
-        cache: 'no-cache', // *default, no-cache, reload, force-cache, only-if-cached
-        credentials: 'same-origin', // include, *same-origin, omit
-        // referrer: '',
-        //referrerPolicy: 'no-referrer',
-        headers: {
-          'Content-Type': 'application/json',
-          // 'Content-Type': 'application/x-www-form-urlencoded',
-        },
-        body: JSON.stringify({
-          refreshToken: this.refreshToken,
-        }), // body data type must match "Content-Type" header
+      const response = await getBackendSrv().datasourceRequest({
+        url: `/api/datasources/${id}/resources/access-token`,
```

Comment:
Sorry! It should be `this.id`. It's the id of the data source.

Reply:
Thank you! I've chaged it, id is defined now. But /api/datasources/15/resources/access-token gives 500 error now with the message "Failed to call resource". Should I build Go code somehow before usage? I have just restarted Grafana and rebuilt Datasource after your update

Reply:
Yes, you need to install [Mage](https://magefile.org/), and then run `mage -v` in the root directory. Note that you need to restart Grafana after building the backend.

Reply:
Done, but looks like unsigned backend plugin is invisible for me, I can't see and test it. 
<img width="936" alt="Screenshot 2021-04-08 at 20 04 23" src="https://user-images.githubusercontent.com/6943577/114067859-f4432900-98a5-11eb-9cb2-1f57477617ed.png">

Reply:
Yes, you need to allow it in the config. In future versions, this will apply for all plugins, not just backend plugins.

https://grafana.com/docs/grafana/latest/administration/configuration/#allow_loading_unsigned_plugins

Reply:
Thank you, I've checked the backend plugin. Looks like it works fine